unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/BoxShape/ECSBoxShapeComponentHandler.cs

```diff
@@ -39,6 +46,7 @@ public void OnComponentModelUpdated(IParcelScene scene, IDCLEntity entity, ECSBo
                 DisposeMesh(scene);
                 generatedMesh = shape.mesh;
                 GenerateRenderer(generatedMesh, scene, entity, model);
+                OnResourceReady?.Invoke(this);
```

Comment:
why can't we have the tracking just in the DataStore and call here something like `DataStore.i.ecs7.SetReady(...)`